### ISSUE TITLE
Level up for Dynamics 365 / PowerApps doesn't open locked fields in GOD mode

### ISSUE THREAD

**Issue by NadiaPer** [rajyraman/Levelup-for-Dynamics-CRM#129]
Recently (a couple of weeks) the GOD mode in 
Level up for Dynamics 365 / PowerApps doesn't open locked fields

Using desktop Chrome. Have admin access rights to the system. Online DynamocsPSA. Model driven app. Consistent for various entities (Account, Opportunity,  Project, and more)

**Comment by rajyraman** [comment]
@NadiaPer - I am not able to repro this on a custom MDF. Can you check in DevTools Console, and see whether any error message is displayed?

**Comment by NadiaPer** [comment]
Thank you, @rajyraman .
We checked DevTools > Console, and didn't find any error messages or reported failures.
Our developer took a further step, and took a peak at the source code.
If our understanding of the source code is correct, extension's God Mode if no longer making read-only fields editable. Which was one of the features previously, and was highly important to our Dynamics Admin team. Was this change intentional, or was it omitted from the solution in the recent update in error?

For reference, here's what we found in the source code:

`

this.utility.Xrm.Page.data.entity.attributes.forEach((a) => a.setRequiredLevel('none'));
--
 
this.utility.Xrm.Page.ui.controls.forEach((c: Xrm.Page.StandardControl) => {
c.setVisible(true);
if (c.setDisabled) {
c.setDisabled(false);
}
if (c.clearNotification) {
c.clearNotification();
}
});
 
this.utility.Xrm.Page.ui.tabs.forEach((t) => {
t.setVisible(true);
t.setDisplayState('expanded');
t.sections.forEach((s) => s.setVisible(true));
});

`

**Comment by rajyraman** [comment]
@NadiaPer -I went through the history and the code has not been changed at all for this one. See https://github.com/rajyraman/Levelup-for-Dynamics-CRM/commit/5fa9f34677b62e36d22beaa0812471a56f883940#diff-a8af030d55e6c631bffdeae725351546. I will try this on a form which has read only fields and check whether it is due to some MSFT update. So, the only issue is readonly fields not made editable?

**Comment by NadiaPer** [comment]
correct. That's the functionality we're missing - being able to write into read-only fields. Thanks

**Comment by rajyraman** [comment]
Can you try it in a different model driven app and see whether you get the same behaviour, as the code has not changed for a while?

**Comment by rajyraman** [comment]
@NadiaPer - Can you please sideload the extension from latest release (https://github.com/rajyraman/Levelup-for-Dynamics-CRM/releases/latest) and see whether it fixes the issue?

**Comment by rajyraman** [comment]
@NadiaPer - Please check if this is still an issue in v3.5.5
